site2/website/release-notes.md

```diff
@@ -1,6 +1,147 @@
+* [Java client] MultiTopics discovery is broken due to discovery task scheduled twice instead of pendingBatchReceiveTask [#6407](https://github.com/apache/pulsar/pull/6407)
+* Fixed publish buffer limit does not take effect [#6431](https://github.com/apache/pulsar/pull/6431)  
+* Fixed the bug of authenticationData is't initialized. [#6440](https://github.com/apache/pulsar/pull/6440)  
+* Fixed the max backoff configuration for lookups [#6444](https://github.com/apache/pulsar/pull/6444)  
+* Fixed avro schema decode error in functions [#6445](https://github.com/apache/pulsar/pull/6445)  
+* pulsar-proxy: fixed correct name for proxy thread executor name [#6460](https://github.com/apache/pulsar/pull/6460)  
+* [pulsar-proxy] fixed logging for published messages [#6474](https://github.com/apache/pulsar/pull/6474)  
+* Fixed create partitioned topic with a substring of an existing topic name. [#6478](https://github.com/apache/pulsar/pull/6478)  
+* Fixed memory leak when running topic compaction. [#6485](https://github.com/apache/pulsar/pull/6485)  
+* [proxy] Fixed proxy routing to functions worker [#6486](https://github.com/apache/pulsar/pull/6486)  
+* [pulsar-client] fixed deadlock on send failure [#6488](https://github.com/apache/pulsar/pull/6488)  
+* [pulsar-client-cpp] Fixed Redelivery of Messages on UnackedMessageTracker When Ack Messages . [#6498](https://github.com/apache/pulsar/pull/6498)  
+* Fixed admin getLastMessageId return batchIndex [#6511](https://github.com/apache/pulsar/pull/6511)  
+* Fixed topic with one partition cannot be updated [#6560](https://github.com/apache/pulsar/pull/6560)  
+* [pulsar-client] Fixed NPE while call getLastMessageId [#6562](https://github.com/apache/pulsar/pull/6562)  
+* Do not retry on authorization failure [#6577](https://github.com/apache/pulsar/pull/6577)  
+* Fixed send get raw schema request [#6650](https://github.com/apache/pulsar/pull/6650)  
+* Fixed avro schema decode error in functions [#6662](https://github.com/apache/pulsar/pull/6662)  
+* [broker]Handle BadVersionException thrown by updateSchemaLocator() [#6683](https://github.com/apache/pulsar/pull/6683)  
+* Fixed restore clusterDispatchRate policy for compatibility [#6176](https://github.com/apache/pulsar/pull/6176)  
+* Fix maven broken link [#6068](https://github.com/apache/pulsar/pull/6068)  
+
+
+#### Enhancements
+
+* Expose bookkeeper expose explicit lac configuration in broker.conf [#5822](https://github.com/apache/pulsar/pull/5822)  
+* [build] Skip javadoc task for pulsar-client-kafka-compact modules [#5836](https://github.com/apache/pulsar/pull/5836)  
+* Add backlogSize in topicStats [#5914](https://github.com/apache/pulsar/pull/5914)  
+* Allow to enable/disable delayed delivery for messages on namespace [#5915](https://github.com/apache/pulsar/pull/5915)  
+* Upgrade Avro to 1.9.1 [#5938](https://github.com/apache/pulsar/pull/5938)  
```

Comment:
Avro 1.9.1 enables the JSR310 datetimes by default, which might introduce some regression problems if users use generated source code by Avro compiler 1.8.x and contains datetimes fields. It's better to use Avro 1.9.x compiler to recompile.